robottelo/hosts.py

```diff
@@ -796,6 +849,15 @@ def nailgun_capsule(self):
 
         return NailgunCapsule().search(query={'search': f'name={self.hostname}'})[0]
 
+    @cached_property
+    def is_upstream(self):
```

Comment:
this would be a good check to add to the Satellite's `version` property

Reply:
done, 04d071b6e